fn_phish_tank/fn_phish_tank/components/fn_phish_tank_submit_url.py

```diff
@@ -0,0 +1,92 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+"""Function implementation"""
+
+import logging
+from resilient_circuits import ResilientComponent, function, handler, StatusMessage, FunctionResult, FunctionError
+import fn_phish_tank.util.selftest as selftest
+from resilient_lib import ResultPayload
+from fn_phish_tank.util.phish_tank_helper import *
+
+
+class FunctionComponent(ResilientComponent):
+    """Component that implements Resilient function 'fn_phish_tank_submit_url"""
+
+    def __init__(self, opts):
+        """constructor provides access to the configuration options"""
+        super(FunctionComponent, self).__init__(opts)
+        self.options = opts.get("fn_phish_tank", {})
+        selftest.selftest_function(opts)
+
+    @handler("reload")
+    def _reload(self, event, opts):
+        """Configuration options have changed, save new values"""
+        self.options = opts.get("fn_phish_tank", {})
+
+    @function("fn_phish_tank_submit_url")
+    def _fn_phish_tank_submit_url_function(self, event, *args, **kwargs):
+        """Function: Function to check URLs against PhishTanks(https://www.phishtank.com/) Database to see URL is flagged as phishing or not phishing."""
```

Comment:
This function checks URLs against PhishTank(https://www.phishtank.com/) database to see if the URL is flagged as Phishing or not Phishing.

Reply:
ok